ArduSub/joystick.cpp

```diff
@@ -115,6 +116,7 @@ void Sub::transform_manual_control_to_rc_override(int16_t x, int16_t y, int16_t
     RC_Channels::set_override(8, lights1, tnow);       // lights 1
     RC_Channels::set_override(9, lights2, tnow);       // lights 2
     RC_Channels::set_override(10, video_switch, tnow); // video switch
+    RC_Channels::set_override(11, strobe_switch, tnow); // strobe switch
```

Comment:
@patrickelectric what do you think about just using a bool flag internal to the vehicle instead of going through the RC channels library?

I know this is what we did for the same thing before (video switch), but it appears to be a very roundabout way of tracking a simple flag. That may have been a poor decision in the past.

Reply:
Yeah, it makes more sense.